lib/client-base.js

```diff
@@ -319,6 +327,11 @@ class Parser extends HTTPParser {
     this.body = null
   }
 
+  [HTTPParser.kOnTimeout] () {
```

Comment:
What happens if `kOnTimeout` is undefined, i.e. node < 14?

Reply:
I see from [here](https://github.com/nodejs/node/blob/v12.18.1/lib/_http_common.js#L47) that a default is always assigned to the key.

On 14 the same is done for the `kOnTimeout` (https://github.com/nodejs/node/blob/v14.4.0/lib/_http_common.js#L50). Maybe we could use the same approach here?